meteor/server/databaseMigration.ts

```diff
@@ -244,7 +247,7 @@ export function runMigration (
 			let stepInput: MigrationStepInputFilteredResult = {}
 			_.each(inputResults, (ir) => {
 				if (ir.stepId === step.id) {
-					stepInput[ir.attribute] = ir.value
+					objectPathSet(stepInput, ir.attribute, ir.value)
```

Comment:
@nytamin THIS I'm a bit afrais of. Basically, any device IDs with `.` in them will break, I guess? Scary